### ISSUE TITLE
[Test Failure] BasicWinForms - Change Accessibility failed in windows_vsi_p1_open_prtest #14889

### ISSUE THREAD

**Issue by genlu** [dotnet/roslyn#14893]
BasicWinForms - Scenario: Change Accessibility failed in #14889 

```
Scenario: Change Accessibility
    Executing action [11/2/2016 3:20:34 PM] AddProject
    Executing action [11/2/2016 3:20:35 PM] OpenFile
DEBUG: found and activated window.
DEBUG: start polling for code window
DEBUG: polling complete
    Executing action [11/2/2016 3:20:35 PM] WinFormsAddControl
        Control 'SomeButton' added successfully.
    Executing action [11/2/2016 3:20:35 PM] WinFormsEditControlProperty
        Property 'Modifiers' changed to 'Public' successfully.
    Executing action [11/2/2016 3:20:35 PM] Build
Waiting for Roslyn features SolutionCrawler: 11/2/2016 3:20:36 PM
Waiting for Roslyn features ErrorList: 11/2/2016 3:20:41 PM
Finished waiting for Roslyn features SolutionCrawler and ErrorList: 11/2/2016 3:20:41 PM
Waiting for error list: 11/2/2016 3:20:43 PM
Finished waiting for error list: 11/2/2016 3:21:15 PM
1 projects failed build. Build output:
1>------ Build started: Project: TestProj, Configuration: Debug Any CPU ------
1>d:\j\workspace\windows_vsi_p---9a06aa34\Open\Binaries\Temp\hxhziubr.ogi\TestProj\Form1.Designer.vb(25,9): error BC30456: 'SomeButton' is not a member of 'Form1'.
1>d:\j\workspace\windows_vsi_p---9a06aa34\Open\Binaries\Temp\hxhziubr.ogi\TestProj\Form1.Designer.vb(30,9): error BC30456: 'SomeButton' is not a member of 'Form1'.
1>d:\j\workspace\windows_vsi_p---9a06aa34\Open\Binaries\Temp\hxhziubr.ogi\TestProj\Form1.Designer.vb(31,9): error BC30456: 'SomeButton' is not a member of 'Form1'.
1>d:\j\workspace\windows_vsi_p---9a06aa34\Open\Binaries\Temp\hxhziubr.ogi\TestProj\Form1.Designer.vb(32,9): error BC30456: 'SomeButton' is not a member of 'Form1'.
1>d:\j\workspace\windows_vsi_p---9a06aa34\Open\Binaries\Temp\hxhziubr.ogi\TestProj\Form1.Designer.vb(33,9): error BC30456: 'SomeButton' is not a member of 'Form1'.
1>d:\j\workspace\windows_vsi_p---9a06aa34\Open\Binaries\Temp\hxhziubr.ogi\TestProj\Form1.Designer.vb(40,25): error BC30456: 'SomeButton' is not a member of 'Form1'.
========== Build: 0 succeeded, 1 failed, 0 up-to-date, 0 skipped ==========

--------------------------------
Tao.Engine.Exceptions.UnexpectedResultException`1[Roslyn.VSHosting.Test.RoslynActions.BuildAction]: 6 Unexpected errors during build:
'SomeButton' is not a member of 'Form1'., File: Form1.Designer.vb, Project: TestProj
'SomeButton' is not a member of 'Form1'., File: Form1.Designer.vb, Project: TestProj
'SomeButton' is not a member of 'Form1'., File: Form1.Designer.vb, Project: TestProj
'SomeButton' is not a member of 'Form1'., File: Form1.Designer.vb, Project: TestProj
'SomeButton' is not a member of 'Form1'., File: Form1.Designer.vb, Project: TestProj
'SomeButton' is not a member of 'Form1'., File: Form1.Designer.vb, Project: TestProj
   at Tao.Utilities.Extensions.ThrowUnexpectedResultException[TAction](TAction action, String message, Object[] messageArgs) in f:\dd\src\Omni\Tao\Tao.Utilities\Extensions.cs:line 465
   at Roslyn.VSHosting.Test.RoslynActions.BuildAction.Execute(TaoEngine engine) in d:\j\workspace\windows_vsi_p---9a06aa34\Closed\Hosting\RoslynTaoActions\Actions\VisualStudio\BuildAction.cs:line 145
   at Tao.Engine.DebuggableActionExecutor.Execute() in f:\dd\src\omni\TAO\Tao.Engine\ActionExecutor.cs:line 139
   at Tao.Engine.DefaultActionExecutor.Execute() in f:\dd\src\omni\TAO\Tao.Engine\ActionExecutor.cs:line 36
================================


Saved screenshot: d:\j\workspace\windows_vsi_p---9a06aa34\Open\Binaries\Release\Exes\EditorTestApp\Screenshots\2016-11-02_15-21-17_27 - Exception Tao.Engine.Exceptions.UnexpectedResultException`1[Roslyn.VSHosting.Test.RoslynActions.BuildAction].png

    Executing action [11/2/2016 3:21:17 PM] CloseSolution
```
![basicwinforms](https://cloud.githubusercontent.com/assets/788783/19940165/f71be784-a0e8-11e6-90ea-5e4b074fc5a0.png)

**Comment by Pilchie** [comment]
Let's use #14833 to track the general failures in the Winforms VSI tests.
